pysat/tests/test_utils.py

```diff
@@ -486,7 +485,7 @@ def test_netcdf_attribute_override(self):
         return
 
 
-class TestLoadNetCDF4XArray(object):
+class TestLoadNetCDF4XArray(TestLoadNetCDF4):
```

Comment:
File output for xarray needs to be fixed before this can use inheritance.  See #60

Reply:
This is going to be performed in this branch, so will be kept in here.  One of the things I'm asking for help with.